internal/server/errors.go

```diff
@@ -41,10 +41,9 @@ func (a *API) sendAPIError(c *gin.Context, err error) {
 		resp.Message = err.Error()
 	case errors.As(err, validationErrors):
 		resp.Code = http.StatusBadRequest
-		resp.Message = err.Error()
+		resp.Message = fmt.Sprintf("%s: %s", internal.ErrBadRequest, err)
```

Comment:
What's the reason for adding "bad request" to this message? Won't we end up with duplicate messages on the client side, because of the 400 status code?

Reply:
Removed this